Source: Gentoo #278394
Upstream: Reported http://dev.mutt.org/trac/ticket/3300
Reason: mutt failed with new gpgme
--- mutt-1.5.20/init.c	2010-01-16 23:16:55.000000000 +0200
+++ mutt-1.5.20/init.c	2010-01-16 23:18:52.000000000 +0200
@@ -50,6 +50,10 @@
 #include <errno.h>
 #include <sys/wait.h>
 
+#if defined(CRYPT_BACKEND_GPGME)
+#include <gpgme.h>
+#endif
+
 #define CHECK_PAGER \
   if ((CurrentMenu == MENU_PAGER) && (idx >= 0) &&	\
 	    (MuttVars[idx].flags & R_RESORT)) \
@@ -3143,6 +3147,11 @@
 
   mutt_read_histfile ();
 
+#if defined(CRYPT_BACKEND_GPGME)
+  /* needed since version 1.2.0, ticket #3300 */
+  gpgme_check_version (NULL);
+#endif
+
 #if 0
   set_option (OPTWEED); /* turn weeding on by default */
 #endif
